doc/source/cluster/index.rst

```diff
@@ -3,227 +3,33 @@
-  Nmap scan report for compute04.berkeley.edu (123.456.78.910)
-  Host is up (0.0011s latency).
-  rDNS record for 123.456.78.910: compute04.berkeley.edu
-  PORT     STATE  SERVICE REASON       VERSION
-  6379/tcp closed redis   reset ttl 60
-  Service detection performed. Please report any incorrect results at https://nmap.org/submit/ .
-  $ nc -vv -z $HEAD_ADDRESS $PORT
-  nc: connect to compute04.berkeley.edu port 6379 (tcp) failed: Connection refused
-
-
-Stopping Ray
-~~~~~~~~~~~~
-
-When you want to stop the Ray processes, run ``ray stop`` on each node.
-
-.. _using-ray-on-a-cluster:
-
-Running a Ray program on the Ray cluster
-----------------------------------------
-
-To run a distributed Ray program, you'll need to execute your program on the same machine as one of the nodes.
-
-.. tabs::
-  .. group-tab:: Python
-
-    Within your program/script, you must call ``ray.init`` and add the ``address`` parameter to ``ray.init`` (like ``ray.init(address=...)``). This causes Ray to connect to the existing cluster. For example:
-
-    .. code-block:: python
-
-        ray.init(address="auto")
-
-  .. group-tab:: Java
-
-    You need to add the ``ray.address`` parameter to your command line (like ``-Dray.address=...``).
+One of Ray's strengths is the ability to leverage multiple machines in the same program. Ray can, of course, be run on a single machine (and is done so often) but the real power is using Ray on a cluster of machines.
 
-    To connect your program to the Ray cluster, run it like this:
+A Ray cluster consists of a **head node** and a set of **worker nodes**. The head node needs to be started first, and the worker nodes are given the address of the head node to form the cluster.
 
-        .. code-block:: bash
+Distributed Ray is powered by the Ray Cluster Launcher, which automatically provisions machines and launches a multi-node Ray cluster. You can use the cluster launcher on GCP, Amazon EC2, Azure, or even Kubernetes. Distributed Ray also makes use of the Ray Autoscaler, which gives Ray the ability "auto-scale," meaning that it can interact with a Cloud Provider to request or release instances according to application workload.
 
-            java -classpath <classpath> \
-              -Dray.address=<address> \
-              <classname> <args>
+How does it work?
+-----------------
 
-    .. note:: Specifying ``auto`` as the address hasn't been implemented in Java yet. You need to provide the actual address. You can find the address of the server from the output of the ``ray up`` command.
+The Ray Cluster Launcher will automatically enable a load-based autoscaler. The autoscaler resource demand scheduler will look at the task, actor, and placement group resource demands from the cluster, and try to add the minimum list of nodes that can fulfill these demands. When nodes are idle for more than :ref:`idle_timeout_minutes <cluster-configuration-idle-timeout-minutes>`, they will be removed. The head node is never removed.
```

Comment:
```suggestion
The Ray Cluster Launcher will automatically enable a load-based autoscaler. The autoscaler resource demand scheduler will look at the pending tasks, actors, and placement groups from the cluster, and try to add nodes that can fulfill these demands. When nodes are idle for more than :ref:`idle_timeout_minutes <cluster-configuration-idle-timeout-minutes>`, they will be removed. The head node is never removed.
```

IMO "resource demands" is actually kinda confusing. Also, we don't want to guarantee these policy related minimums since they're not necessarily true or a good idea.

Reply:
Added to the commit